oaknational/Oak-Web-Application
# fix(secret-manager): get config+secrets working with google secret ma…

## Description

- Add env to bypass network secret fetching
- Add documentation for secrets

## Issue(s)

Re #204

## How to test

Locally, add env:
```
GOOGLE_SECRET_MANAGER_SERVICE_ACCOUNT="1password > GCP Credentials: Vercel Web Build User"
USE_ONLY_LOCAL_SECRETS=true
```
1. run `npm run dev`
2. should successfully start app
3. comment out `USE_ONLY_LOCAL_SECRETS
4. should fail to start app (with permissions error -- this is todo)
5. comment out GOOGLE_SECRET_MANAGER_SERVICE_ACCOUNT
6. should fail to start app (`Error: Please set env for GOOGLE_SECRET_MANAGER_SERVICE_ACCOUNT`)
